Sources/Kitura/Router.swift

```diff
@@ -126,14 +126,22 @@ public class Router {
 
     // MARK: Template Engine
 
-    /// Register a templating engine. The templating engine will handle files in the `viewsPath`
-    /// that match the file extension it supports.
+    /// A template engine allows rendering of documents using static templates.
```

Comment:
Convention in the other functions is that the first thing you see explicitly says what the specific function does e.g. "Set up a sub router to handle requests." With any extra explanation coming after. I think if you just move the first comment line from your code (i.e. "Register a template engine to a given router instance") to the top here it's fine 🙂

Reply:
I'll look into how to best phrase this as 'Register a templating engine' just felt out of place there. I like your idea of moving the comment up to the top, that could work with a little rewording :).